ruleset-style.md

```diff
@@ -69,7 +75,7 @@ Here is an example ruleset pre-style guidelines:
 
 ```
 <ruleset name="WHATWG.org">
-  <target host="whatwg.org" />
+  <target host='whatwg.org' />
```

Comment:
This is made to illustrate how a rule may look _before_ the styleguide changes are applied.

One of the styleguide changes are to consistently use double quotes.  This is meant to illustrate that change.